solidity/dashboard/src/utils/general.utils.js

```diff
@@ -74,3 +74,17 @@ export const formatPercentage = (value) => {
 
   return value.decimalPlaces(2, BigNumber.ROUND_DOWN).toNumber()
 }
+
+export const displayPercentageValue = (value, isFormattedValue = true) => {
+  if (!isFormattedValue) {
+    value = formatPercentage(value)
+  }
+
+  let prefix = ""
+  if (value === 0.01) {
+    prefix = `<`
+  } else if (value >= 999) {
+    prefix = `>`
+  }
+  return `${prefix}${value}%`
+}
```

Comment:
Maybe we could actually set min and max values as arguments of the function? And then we could just set the default values for them (0.01 and 999). This would make the function more reusable in the future, because we could set up the range, for which we display the prefixes, by passing the right arguments.

In this case we could probably also change `value === 0.01` to `value <= 0.01`